### PR TITLE
Support alpha transparency in geom_area

### PR DESCRIPTION
**Pull request by mkcor**
Small PR addressing issue #162 -- will update version number after review.

/cc @tdhock please :)

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+
+Support colour aesthetic in geom_text().
+
+0.5.15 -- 19 December 2014.
+
+Use proper RCurlOptions in get_figure() method.
+
+0.5.14 -- 1 December 2014.
+
+Make layers geom_line + geom_point only one trace in Plotly.
```
Comment (tdhock) [inline NEWS]: I re-implemented this functionality in toby-fixes branch, at the end of gg2list.

**Comment by tdhock** [comment]
There are a lot of commits so I am not sure I understand them all, but I had a brief look and I think there is only 1 thing that overlaps with my commits in the toby-fixes branch. That is the merging of lines, points, and error bars to a single trace, at the end of gg2list.

Anyway I think the best way to go forward will be to first merge this branch with master, and then I will fix any conflicts with my toby-fixes branch later.

+1

File: NEWS (hunk 1 above)
Comment (mkcor) [inline NEWS]: Good to know.

**Comment by mkcor** [comment]
Sorry, you shouldn't have seen very many commits, only a few...  I made a mistake when working on the image tests.  Here you go: https://github.com/ropensci/plotly/commit/84a9cd869b54c14188573c16abe3b228955967b5
/cc @chriddyp

**Pull request merged**